examples/natural_language_classifier_v1.py

```diff
@@ -18,8 +18,12 @@
 
 # create a classifier
```

Comment:
is there a reason why these are commented out? are the examples run with some sort of CI?

Reply:
@germanattanasio ☝️

Reply:
Yes, the examples serve double-duty as they are used as integration tests.